pkg/deploy/deploy_test.go

```diff
@@ -28,26 +31,48 @@ func TestGetParameters(t *testing.T) {
 			},
 		},
 		{
-			name: "valid",
+			name: "all parameters present",
```

Comment:
Function overall populates templates parameters from config.
`ps` is template aka. required values.
`config` is there those values should be stored. 

So this test is testing "all configuration values are present". I think make of the test is confusing (it was confusing before too)